pyproject.toml

```diff
@@ -15,6 +15,7 @@ graphene-django = "^2"
 django-graphql-jwt = "^0.3.0"
 PyJWT = "^1.7.0"
 django-cors-headers = "^3.7.0"
+flake8 = "^3.9.2"
```

Comment:
Do you think we can move `flake8` to the dev-dependencies?

We usually keep it there in other grimoirelab repositories.
https://github.com/chaoss/grimoirelab-perceval/blob/master/pyproject.toml#L55